tests/platform_tests/api/test_psu.py

```diff
@@ -32,10 +32,19 @@
 STATUS_LED_COLOR_RED = "red"
 STATUS_LED_COLOR_OFF = "off"
 
+@pytest.fixture(scope="class")
+def gather_facts(request, duthost):
+    # Get platform facts from platform.json file
+    request.cls.chassis_facts = duthost.facts.get("chassis")
+    if not request.cls.chassis_facts:
+        logger.warning("Unable to get chassis_facts from platform.json, test results will not be comprehensive")
```

Comment:
Remove warning message and make the check mandatory. See SFP portion of https://github.com/Azure/sonic-mgmt/pull/2268 for examples.